inputs/invalid_typecheckGuest.cal

```diff
@@ -0,0 +1,17 @@
+Start
+
+Guests: 
+JC is Jeremy Chiu with jreggie@gmail.com
+Lydia is Lydia Sweating Choi with lydiachoi227@somedomain.org
+Chris is Chris Bolton with chris@bolton.ca
+Done
+
+Locations: 
+location B is "2053 Main Mall, Vancouver, BC V6T 1Z2"
+Starbucks is "Starbucks on main, 2980 Main St, Vancouver, BC"
+Done
+
+Events:
+"PARTY" on October 10, 2019 from 12:30 to 13:00 at starbucks with starbucks and Chris 
```

Comment:
I think the namecheck/typecheck discrepancy is occuring because these are lower case variable names `starbucks`, but the defined variable `Starbucks` is titlecased.